resources/typing-dna/plugins/typing-dna.jsp

```diff
@@ -30,7 +30,7 @@
 
 <script>
     var tdna = new TypingDNA();
-    tdna.addTarget("username");
+    typingdna.addTarget("usernameUserInput");
```

Comment:
shouldn't this be tdna.addTarget ?

Reply:
fixed in https://github.com/wso2-extensions/identity-conditional-auth-typingdna/pull/29/commits/65b5b86f62c1de285f040522c919402480f6dd26